src/main/java/nextstep/optional/User.java

```diff
@@ -1,69 +1,74 @@
 package nextstep.optional;
 
+import java.util.Optional;
+
 public class User {
-    private String name;
-    private Integer age;
+	private String name;
+	private Integer age;
 
-    public User(String name, Integer age) {
-        this.name = name;
-        this.age = age;
-    }
+	public User(String name, Integer age) {
+		this.name = name;
+		this.age = age;
+	}
 
-    public String getName() {
-        return name;
-    }
+	public String getName() {
+		return name;
+	}
 
-    public Integer getAge() {
-        return age;
-    }
+	public Integer getAge() {
+		return age;
+	}
 
-    public boolean matchName(String name) {
-        return this.name.equals(name);
-    }
+	public boolean matchName(String name) {
+		return this.name.equals(name);
+	}
 
-    public static boolean ageIsInRange1(User user) {
-        boolean isInRange = false;
+	public static boolean ageIsInRange1(User user) {
+		boolean isInRange = false;
 
-        if (user != null && user.getAge() != null
-                && (user.getAge() >= 30
-                && user.getAge() <= 45)) {
-            isInRange = true;
-        }
-        return isInRange;
-    }
+		if (user != null && user.getAge() != null
+				&& (user.getAge() >= 30
+				&& user.getAge() <= 45)) {
```

Comment:
```suggestion
                            && optionalUser.범위검증()
```
객체에게 범위 검증하라고 시켜봐도 좋을것 같아요!